src/cron/schedule-event-reminder.js

```diff
@@ -19,6 +19,24 @@ const clearAllCronJobs = () => {
   activeCronJobs = [];
 };
 
+const generateReminderCronExpression = (event) => {
+  const now = new Date();
+  const eventEndTimeDate = new Date(event.scheduledEndTimestamp);
+  const timeDifference = eventEndTimeDate - now;
+
+  const reminderTime = new Date(
+    timeDifference > dayInMilliseconds
+      ? eventEndTimeDate - dayInMilliseconds
+      : eventEndTimeDate - hoursInMilliseconds
```

Comment:
maybe dayInMilliseconds should be in uppercase as it is a constant:

DAYS_IN_MILLISECONDS
HOURS_IN_MILLISECONDS

This way reading the code we can understand that it's value is not changing